cropper/src/main/java/com/theartofdev/edmodo/cropper/CropImageView.java

```diff
@@ -512,6 +513,22 @@ public void setGuidelines(Guidelines guidelines) {
     return new Pair<>(mCropOverlayView.getAspectRatioX(), mCropOverlayView.getAspectRatioY());
   }
 
+  /** both the X and Y values of the aspectRatio of the image to be cropped. */
+  public Pair<Integer, Integer> getImageAspectRatio() {
+    if (mBitmap != null) {
+      BigInteger b1 = BigInteger.valueOf(mBitmap.getWidth());
```

Comment:
why use `BigInteger`?

Reply:
just wanted to use the BigInteger.gcd() function. couldn't find a built-in one for Integers